src/main/resources/scss/abstracts/_typography.scss

```diff
@@ -1,21 +1,41 @@
 .jdl-heading {
   font-weight: 600;
   font-size: 1.2rem;
-  margin: calc(var(--section-padding) / 2) 0;
+  margin: calc(var(--jdl-spacing) / 2) 0;
 }
 
 .jdl-leading-paragraph {
-  font-size: 1rem;
+  font-size: 1.2rem;
+  font-weight: 600;
+  margin: 0 0 var(--jdl-spacing) 0;
+  line-height: 1.66;
+
+  & + & {
+    margin-top: calc((var(--jdl-spacing) / 2) * -1);
+  }
+}
+
+.jdl-paragraph {
+  font-size: 1.1rem;
   font-weight: 500;
-  margin: 0 0 var(--section-padding) 0;
+  margin: 0 0 var(--jdl-spacing) 0;
+  line-height: 1.66;
 
   & + & {
-    margin-top: calc((var(--section-padding) / 2) * -1);
+    margin-top: calc((var(--jdl-spacing) / 2) * -1);
   }
 }
 
+.jdl-list {
+  font-size: 1.1rem;
+  font-weight: 500;
```

Comment:
is this font-weight needed?

it seems a bit much in:

<img width="703" alt="image" src="https://user-images.githubusercontent.com/21194782/171050046-82a98d1c-02a5-4ee8-b22b-2b9b1e3f2fd7.png">

Reply:
Are you on Chrome? I've just opened a PR (that you've just approved :P) https://github.com/jenkinsci/jenkins/pull/6614#pullrequestreview-989694771 to address the weird font issues on there. Fonts should have better weighting now on Chrome, be that if they're light or bold.

https://user-images.githubusercontent.com/43062514/171062827-c318c45c-abf3-4159-a04d-7fb73a20654c.mov

Reply:
Yes on chrome

Reply:
Is the new font rendering better for you now?

Reply:
Still a bit bold for regular text I think,

<img width="938" alt="image" src="https://user-images.githubusercontent.com/21194782/171113107-e574c752-03a1-4528-8f9a-1e237ffdbb65.png">